src/domain/local-storage/index.ts

```diff
@@ -1,4 +1,9 @@
-import { PendingDeposit, PendingWithdraw, PendingDelayedWithdraw } from "src/domain/hermez";
+import {
+  PendingDeposit,
+  PendingWithdraw,
+  PendingDelayedWithdraw,
+  AvailableWithdraw,
```

Comment:
Maybe call it TimerWithdraw or CheckAvailabilityWithdraw to make it clear what it is. Otherwise, it could be all exits that are available to withdraw

Reply:
I like TimerWithdraw